assets/common.css

```
@@ -22,6 +22,13 @@ code {
   font-size: 0.8rem;
 }
 
+pre {
+  /* Need white background for color contrast */
+  background-color: white !important;
```

```suggestion
:not(pre)>code[class*=language-], pre[class*=language-] {
  /* Need white background for color contrast */
  background-color: white;
```
Would prefer to avoid using `!important` in declarations as it makes it harder to manage CSS. The background color for `<pre>` elements here is being driven by prism.css. We can either override the selector in common.css or edit assets/prism.css directly, as it is part of the repo.

---

I'm assuming prism.css is an external library that would be updated if we were to do any package maintenance so we should avoid editing that -- let me know if this isn't the case.